gradle.properties

```diff
@@ -1,5 +1,5 @@
 # production library version numbers
-toolingApiVersion=8.6
+toolingApiVersion=8.9-20240605001727+0000
```

Comment:
This will be a released version before merge, right?

Reply:
Yes, I want to merge the feature only with a final release version of the tapi.